01_dask.delayed.ipynb

```diff
@@ -243,10 +243,23 @@
   {
```

Comment:
<p>It doesn't show up well in nbsphinx, but this cell is hidden in Jupyterlab. Not sure what it will look like in the rendered version, but it'll probably be visible, and that's probably what we want.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dask/dask-tutorial/pull/144/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>